### PR TITLE
[test-api-update] dagster/storage_tests

### PR DESCRIPTION
**Pull request by smackesey**
## Summary & Motivation

Convert dagster_tests/storage_tests fully over to job/op/graph APIs.

## How I Tested These Changes

BK

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13145** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13145" target="_blank">Graphite</a> 
    * **PR #13108** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13108" target="_blank">Graphite</a> 
      * **PR #13168** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13168" target="_blank">Graphite</a> 
        * **PR #13169** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13169" target="_blank">Graphite</a> 
          * **PR #13081** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13081" target="_blank">Graphite</a> 
            * **PR #13083** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13083" target="_blank">Graphite</a> 
            * **PR #12770** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12770" target="_blank">Graphite</a> 
              * **PR #12769** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12769" target="_blank">Graphite</a> 
              * **PR #13170** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13170" target="_blank">Graphite</a> 
                * **PR #13171** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13171" target="_blank">Graphite</a> 
                  * **PR #13188** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13188" target="_blank">Graphite</a> 
                    * **PR #13172** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13172" target="_blank">Graphite</a> 
                      * **PR #13192** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13192" target="_blank">Graphite</a> 
                        * **PR #13301** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13301" target="_blank">Graphite</a> 
                        * **PR #13295** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13295" target="_blank">Graphite</a> 
                          * **PR #13296** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13296" target="_blank">Graphite</a> 
                            * **PR #13294** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13294" target="_blank">Graphite</a> 
                              * **PR #13297** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13297" target="_blank">Graphite</a> 
                                * **PR #13303** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13303" target="_blank">Graphite</a> 
                                  * **PR #13299** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13299" target="_blank">Graphite</a> 
                                    * **PR #13298** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13298" target="_blank">Graphite</a> 
                                      * **PR #13302** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13302" target="_blank">Graphite</a> 
                                        * **PR #13311** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13311" target="_blank">Graphite</a> 
                                          * **PR #13312** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13312" target="_blank">Graphite</a> 
                                            * **PR #13316** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13316" target="_blank">Graphite</a> 
                                              * **PR #13317** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13317" target="_blank">Graphite</a>  👈
                      * **PR #13173** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13173" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13317?utm_source=stack-comment).

File: python_modules/dagster/dagster_tests/storage_tests/test_io_managers_with_graphs.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def load_input(self, context):
     return my_io_manager
 
 
-def test_composite_solid_output():
+def test_composite_op_output():
```
Comment (dpeng817) [inline python_modules/dagster/dagster_tests/storage_tests/test_io_managers_with_graphs.py:26]: [1] I believe a `composite_op` is just a `graph`, no?

File: python_modules/dagster/dagster_tests/storage_tests/test_io_managers_with_graphs.py [hunk 2]
```diff
@@ -50,16 +50,16 @@ def my_job():
...
+    # manager of my_op_takes_input, not my_composite.
     assert storage_dict[(result.run_id, "my_graph.my_op_takes_input", "result")]["value"] == {
         "value": 5,
         "output_manager_name": "inner",
         "input_manager_name": "inner",
     }
 
 
-def test_composite_solid_upstream_output():
+def test_composite_op_upstream_output():
```
Comment (dpeng817) [inline python_modules/dagster/dagster_tests/storage_tests/test_io_managers_with_graphs.py:62]: [1]

**Review by dpeng817 — APPROVED** [review]
rename `composite_op` stuff to graph

**Pull request merged**